cob_twist_controller/src/inverse_jacobian_calculations/inverse_jacobian_calculation.cpp

```diff
@@ -143,7 +144,7 @@ Eigen::MatrixXd PInvDirect::calculate(const TwistControllerParams& params,
         ROS_ERROR("PInvDirect does not support SVD. Use PInvBySVD class instead!");
     }
 
-    double lambda = db->getDampingFactor(Eigen::VectorXd::Zero(1, 1), jacobian);
+    Eigen::MatrixXd lambda = db->getDampingFactor(Eigen::VectorXd::Zero(1, 1), jacobian);
```

Comment:
as the matrix lambda now already contains the squared damping factor, this has to be considered in the computation below by using ```lambda * ident;``` instead of ```lambda * lambda * ident;```